config/initializers/menus.rb

```diff
@@ -54,6 +54,21 @@
               (User.current.logged? || !Setting.login_required?) &&
                 User.current.allowed_to_globally?(:view_news)
             }
+
+  # TODO: This should be put in modules/meeting/../engine.rb, but if it is put
+  # in there, it causes a NameError because `Setting` that is used by FeatureDecisions
+  # is not loaded yet. So, let's see what to do here...
+  if OpenProject::FeatureDecisions.more_global_index_pages_active?
```

Comment:
You can move the if into the `if:` Proc below, that means the menu is registered in code but never loaded.

Reply:
Yeah, wrote that in https://github.com/opf/openproject/pull/12641#discussion_r1196372340. I'll refactor to do that

Reply:
Done ✅